tardis/energy_input/tests/test_gamma_ray_grid.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import numpy.testing as npt
+import numpy as np
+from astropy.coordinates import cartesian_to_spherical
+
+from tardis.energy_input.gamma_ray_grid import (
+    calculate_distance_radial,
+    distance_trace,
+    move_gamma_ray,
+    density_sampler,
+)
+
+
+@pytest.mark.xfail(reason="To be implemented")
+def test_calculate_distance_radial():
+    assert False
+
+
+@pytest.mark.xfail(reason="To be implemented")
+def test_distance_trace():
+    assert False
+
+
+def test_move_gamma_ray(basic_gamma_ray):
+    """
+
+    Parameters
+    ----------
+    basic_gamma_ray : GammaRay object
+    """
+    gamma_ray = basic_gamma_ray
```

Comment:
we should test easy cases (or edge cases) - move along a direction - move all directions - rather than all